src/routes/+page.server.js

```diff
@@ -19,28 +19,40 @@ export async function load() {
 
 	const hygraphData = await hygraph.request(query);
 
-  
-  const space = "%20";
-  const bookItems = "boeken";
-  const EbookItems = "e-books";
-  const audioItems = "luisterboeken";
-  const urlSearch = "search/";
-  const urlBase = "https://zoeken.oba.nl/api/v1/";
-  const urlQuery = "?q=";
-  const urlDefault = "special:all";
-  const urlKey = `&authorization=${publicKey}`;
-  const urlOutput = "&refine=true&output=json";
-  const defaultUrlBooks = urlBase + urlSearch + urlQuery + urlDefault + space + bookItems + urlKey + urlOutput;
-  const defaultUrleBooks = urlBase + urlSearch + urlQuery + urlDefault + space + EbookItems + urlKey + urlOutput;
-  const defaultUrlAudioBooks = urlBase + urlSearch + urlQuery + urlDefault + space + audioItems + urlKey + urlOutput;
-  
-	const response = await fetch(
-		defaultUrlBooks && defaultUrleBooks && defaultUrlAudioBooks
+
+	const space = "%20";
+	const bookItems = "boeken";
+	const EbookItems = "e-books";
+	const audioItems = "luisterboeken";
+	const urlSearch = "search/";
+	const urlBase = "https://zoeken.oba.nl/api/v1/";
+	const urlQuery = "?q=";
+	const urlDefault = "special:all";
+	const urlKey = `&authorization=${publicKey}`;
+	const urlOutput = "&refine=true&output=json";
+	const defaultUrlBooks = urlBase + urlSearch + urlQuery + urlDefault + space + bookItems + urlKey + urlOutput;
+	const defaultUrleBooks = urlBase + urlSearch + urlQuery + urlDefault + space + EbookItems + urlKey + urlOutput;
+	const defaultUrlAudioBooks = urlBase + urlSearch + urlQuery + urlDefault + space + audioItems + urlKey + urlOutput;
+
+	const responseBooks = await fetch(
+		defaultUrlBooks
+	);
+
+	const responseEBooks = await fetch(
+		defaultUrleBooks
+	);
+
+	const responseAudioBooks = await fetch(
+		defaultUrlAudioBooks
 	);
-	const searchData = await response.json();
+	const apiBooks = await responseBooks.json();
+	const apiAudioBooks = await responseAudioBooks.json();
+	const apiEBooks = await responseEBooks.json();
 
 	return {
 		hygraphData,
-		searchData
+		apiBooks,
+		apiAudioBooks,
+		apiEBooks
 	};
 }
```

Comment:
Super dat de data eindelijk ingeladen kan worden! Kan je er wat comments in zetten zodat we over een paar dagen nog steeds goed snappen wat precies wat is. Naamgeving is trouwens heet duidelijk 👍